resources/uk/gov/hmcts/helm/push-helm-charts-to-git.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+set -x
+
+CHART_DIRECTORY=${1}
+CHART_NAME=${2}
+VERSION=${3}
+
+RETRIES=3
+DELAY=5
+COUNT=0
+
+if [ -d "hmcts-charts" ]; then
+  rm -rf hmcts-charts
+fi
+
+while [ $COUNT -lt $RETRIES ]; do
+  if git clone https://github.com/hmcts/hmcts-charts.git; then
+    COUNT=0
+    break
+  fi
+  COUNT=$(( $COUNT + 1))
+  sleep $DELAY
+done
+
+if cd hmcts-charts; then
+  if [ -d "${CHART_NAME}" ]; then
+    GIT_CHART_VERSION=$(helm inspect chart "${CHART_NAME}" | grep ^version | cut -d  ':' -f 2 | tr -d '[:space:]')
+    if [ "${GIT_CHART_VERSION}" = "${VERSION}" ]; then
+      echo "No differences in the charts. Nothing to publish" 1>&2
```

Comment:
The chart should be published in all case if there are any differences. In that case we should probably increase the version. Or are you counting on a previous script execution to increase the version if there are changes?